tests/admin_views/test_autocomplete_view.py

```diff
@@ -185,6 +185,8 @@ def test_select(self):
         # Load next page of results by scrolling to the bottom of the list.
         for _ in range(len(results)):
             search.send_keys(Keys.ARROW_DOWN)
+        # Wait for ajax delay.
+        time.sleep(0.25)
```

Comment:
Is this the best way to handle this? (cc @codingjoe)

Reply:
No, really no. The assertions in the bottom should be replaced by waits until the result have the right length.
Sleeping will just slow down the test suite artificially and could still fail on some systems.

This is also part of my pull-request. I am not so sure about this ticket. A lot more tests fail for me locally when I run against FireFox. I believe we should consider running selenium tests all the time, as suggested in my GitHub actions patch, to make sure we a stable reference.

Reply:
Grrr... I'd much prefer an explicit wait (for a known condition) then keeping the straight asserts...

Reply:
> A lot more tests fail for me locally when I run against FireFox.

Can you provide the details of these failures? With this patch, running the test suite with `--selenium firefox` results in no failures for me locally.

Reply:
Thanks!

I've removed the `time.sleep()` call here and elsewhere in this file by adding a new `.wait_for_number_of_elements()` method. I've left this as a separate commit in case we'd like to move it to a separate PR but if the preference is to squash it here, just let me know.